src/components/topElements/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { TopElementsProps } from "../../utils/interfaces";
+import './styles.css'
+
+const TopElements = ({getPurchase = (): void => {}}: TopElementsProps): JSX.Element => {
+    const [place, setPlace] = useState<string>('')
+    const [price, setPrice] = useState<number>()
+
+    const getDate = () => {
+        const today = new Date()
+        let month: number | string = today.getMonth() + 1;
+        if (month < 10){ month = '0' + month }
+        return (today.getDate() + '.' + month + '.' + today.getFullYear());
+    }
```

Comment:
This method has nothing to do with the component, so it should be in the utils directory